### PR TITLE
LPS-139738 Display date in a user-friendly format on display page fragments

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-139738

### CODE REVIEW

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:relevant

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]
Hey @p-albuquerque , I thinks it's not a good idea make this change here, because the callers expect to receive the map with field's name as a key and the value itself in their own type. So it's better to keep returning Date object.

Probably you will need to get the Date object and format it in places you need it formatted.

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java [hunk 1]
```diff
@@ -248,6 +254,15 @@ else if (Validator.isNotNull(objectField.getRelationshipType())) {
 				return objectEntry.getTitleValue();
 			}
 		}
+		else if (Objects.equals(objectField.getType(), "Date")) {
+			Locale locale = LocaleThreadLocal.getThemeDisplayLocale();
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java:258]: Hey @p-albuquerque , we already had the locale here using serviceContext.getLocale(), so it's better to delete this line and use this way in the next line.

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java [hunk 2]
```diff
@@ -248,6 +254,15 @@ else if (Validator.isNotNull(objectField.getRelationshipType())) {
 				return objectEntry.getTitleValue();
 			}
 		}
+		else if (Objects.equals(objectField.getType(), "Date")) {
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFieldValuesProvider.java:257]: You will also need to include this logic in class com.liferay.object.web.internal.info.item.renderer.ObjectEntryRowInfoItemRenderer#_getValues, as it is used to display the Object information when a table template is configured in List Style property on Collection Display fragment.

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
Just started reviewing :)

**Comment by guilhermedcamacho** [comment]
Resent on #261

**Pull request closed without merging**